packages/@aws-cdk/aws-appmesh/lib/virtual-node.ts

```diff
@@ -298,25 +280,31 @@ function renderAttributes(attrs?: {[key: string]: string}) {
 /**
  * Interface with properties ncecessary to import a reusable VirtualNode
  */
-interface VirtualNodeAttributes {
+export interface VirtualNodeAttributes {
   /**
    * The name of the VirtualNode
+   *
+   * @default - required when virtualNodeArn is not specified
    */
   readonly virtualNodeName?: string;
 
   /**
-   * The Amazon Resource Name belonging to the VirtualNdoe
+   * The Amazon Resource Name belonging to the VirtualNode
+   *
+   * @default - required when virtualNodeName and mesh are not specified
    */
   readonly virtualNodeArn?: string;
```

Comment:
Same comment here (remove `virtualNodeArn` from `VirtualNodeAttributes`, make `virtualNodeName` and `mesh` required).